docs/advanced/persistence-and-collaboration.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Persistence and Collaboration
+---
+
+By default `Variable` state is purely client-side - by default its source of truth is the browser memory.
```

Comment:
I think it could be nice to add a small sentence or two just about what this page is about. For if someone is flicking through our docs and wants to have a little summary before committing to reading the whole page.

Reply:
I mean the sentences here were meant to be a summary, it's a bit lengthy but it's tricky to explain. How does the current version sound?

Reply:
I realise I made it even longer but I wanted to paint a picture of why this is useful

Reply:
Looks better! How about, instead of:
In order to make the aforementioned default behaviour more customizable to suit different needs, `Variable` accepts a `store` property. It can be used to customize the source of truth for a given `Variable` and enables
persistence and collaboration.

Something like:

To address these limitations, `Variable` accepts a `store` property. This allows for customization of the Variable's source of truth, thereby enabling features like data persistence and collaborative interactions across users.

Reply:
I wouldn't call it a limitation, it's just how it works 😅 Will try to find a better word, but sure looks better